setup.py

```diff
@@ -39,12 +39,18 @@
 VSC_INSTALL_REQ_VERSION = '0.9.19'
 
 PACKAGE = {
-    'version': '2.4.18',
+    'version': '2.5.0',
```

Comment:
I assumed you were using [semantic versioning][1].  Just tell me what version number you would like to have here.

[1]: http://semver.org/